doc/integrations/duckdb-native.md

```diff
@@ -186,6 +167,45 @@ ax = avg_trip_distance.plot()
 _ = ax.set_title("Avg trip distance by num of passengers")
 ```
 
+## Load sample data from a SQLite database
+
+If you have a large SQlite database, you can use DuckDB to perform analytical queries it with much better performance.
+
+```{code-cell} ipython3
+%load_ext sql
+```
+
+```{code-cell} ipython3
+import urllib.request
+from pathlib import Path
+
+# download sample database
+if not Path("my.db").is_file():
+    url = "https://raw.githubusercontent.com/lerocha/chinook-database/master/ChinookDatabase/DataSources/Chinook_Sqlite.sqlite"  # noqa
+    urllib.request.urlretrieve(url, "my.db")
+```
+
+We'll use `sqlite_scanner` extension to load a sample SQLite database into DuckDB:
+
+```{code-cell} ipython3
+import duckdb
+
+conn = duckdb.connect()
+%sql conn
+```
+
+```{code-cell} ipython3
+%%sql
+INSTALL 'sqlite_scanner';
+LOAD 'sqlite_scanner';
+CALL sqlite_attach('my.db');
```

Comment:
Should we pass `--alias` here also?
<img width="822" alt="Screenshot 2023-07-31 at 12 52 39 PM" src="https://github.com/ploomber/jupysql/assets/29096135/db991e12-c7c2-4c91-9939-ad7710dde9aa">